### DISCUSSION TITLE
Adding workspace to existing project on jobs server

### DISCUSSION THREAD

**Discussion by hjforbes** [opensafely/documentation#346] [Q&A] [answered]
I would like to add a repo to the Project •School Age Children on the jobs server. When I select add a new workspace, I can register a new project but can't seem to add a workspace to an existing project.  I am following the documentation here: https://docs.opensafely.org/job-server/.  

Should I just create a new project?

Thanks
Harriet

**Comment by iaindillingham** [comment]
Hello @hjforbes,

Thanks for using OpenSAFELY! Sorry that you've had an issue. There isn't a "Create a new workspace and add it to an existing project" button, unfortunately. However, if you visit <https://jobs.opensafely.org/lshtm/school-age-children/new-workspace/>, then you can do so.

All the best,

Iain

**Comment by ghickman** [comment, marked as answer]
Hi @hjforbes,

Unfortunately this was a small bug in the system which removed the button you should have been seeing!  This should be restored now, apologies for the disruption.

Thanks
George
